pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/endpoints/Endpoint.java

```diff
@@ -4,5 +4,20 @@
 import com.pubnub.api.v2.PNConfiguration;
 
 public interface Endpoint<T> extends ExtendedRemoteAction<T> {
+    /**
+     * Allows to override certain configuration options (see {@link com.pubnub.api.v2.BasePNConfigurationOverride.Builder}) for this request only.
+     * <p>
+     * {@link com.pubnub.api.v2.PNConfigurationOverride#from(com.pubnub.api.v2.BasePNConfiguration)} should be used to obtain a <code>PNConfigurationOverride.Builder</code>.
+     * Only options present in <code>PNConfigurationOverride.Builder</code> will be used for the override.
+     * <p>
+     * Example:
+     * <pre>
+     * configOverride = PNConfiguration.from(pubnub.configuration)
```

Comment:
I can't find integtest that contains PNConfiguration.from
I would add one.
I found in test:
PNConfiguration overrideConfig = PNConfigurationOverride.from(pubNub.getConfiguration()).setUserId(new UserId(expectedUser)).build();

but this is not `PNConfiguration.from` as stated in KDoc above

Reply:
the kdoc is wrong, it should be PNConfigurationOverride, I'll change it